### PR TITLE
Improved Pro Serve description

### PR DESCRIPTION
**Pull request by pete0emerson**
This closes https://github.com/gruntwork-io/company/issues/192

### CODE REVIEW

File: pages/careers/_open-positions.html [hunk 1]
```diff
@@ -114,7 +114,7 @@ <h2>Software Engineering (Pro Serve)</h2>
     <h3 style="margin-top: 5px; margin-bottom: 5px;">Software Engineer (Pro Serve)</h3>
     <h3 style="margin-top: 5px; margin-bottom: 5px;">Senior Software Engineer (Pro Serve)</h3>
     <h4>What You'll Work On</h4>
-    <p>The Pro Serve team’s mission is to help Gruntwork customers be successful with our products.
+    <p>The Professional Services team’s mission is to help Gruntwork customers be successful with our infrastructure products.
```
Comment (brikis98) [inline pages/careers/_open-positions.html:117]:
```suggestion
    <p>The Professional Services (Pro Serve) team’s mission is to help Gruntwork customers be successful with our infrastructure products.
```

File: pages/careers/_open-positions.html [hunk 2]
```diff
@@ -156,13 +156,12 @@ <h4>What You'll Work On</h4>
 
     <h4>Your Ideal Background</h4>
     <ul>
-      <li>You know how to write code across the stack ("Dev").</li>
-      <li>You have experience running production software ("Ops").</li>
+      <li> You have a strong background in software engineering, with 5+ years of experience.</li>
```
Comment (brikis98) [inline pages/careers/_open-positions.html:159]: NIT: extra leading space.

**Review by brikis98 — DISMISSED** [review]
LGTM, just two NITs

**Comment by pete0emerson** [comment]
I've applied the NITs to Workable, once this is approved I'll push it out and then have Amanda send out the tweet.

**Review by brikis98 — APPROVED** [review]
LGTM!

**Pull request merged**